### PR TITLE
2nd js project

### PR DESCRIPTION
**Pull request by S-BLH**
- Created a repository by the name of palindrome-checker.

- cloned it to my local machine.

- changed the branch from main to development and then feature-branch.

- created index.html, style.css and script.js files.

- configured the linters.

- added the necessary code to complete the project.

### CODE REVIEW

**Comment by latifa-wakili** [comment]
good job !
you created a complate repository
your code are clean
no errors
great arranged branchs

**Comment by ahzamir** [comment]
@S-BLH 

- I can see you pushed lots of codes in just a few commits, which is wrong.
- At least have some 5-6 more commits so I can review your project.
  - [SUGESSION] - You can add some more design to your code and commit it.

**Comment by S-BLH** [comment]
@ahzamir as you said I added design to my project, kindly review it.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+
+    button:hover {
+      background-color: darkgreen;
+    }
+
+    #output {
+      font-size: 24px;
+      margin-top: 20px;
+    }
+  </style>
```
Comment (ahzamir) [inline index.html]: - You need to have your CSS code in a separate file like `styles.css` and add its link using `<link>` tag inside your `<head>` tag in your HTML file.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+      for (let i = 0; i < numeralMap.length; i++) {
+        while (number >= numeralMap[i].value) {
+          roman += numeralMap[i].numeral;
+          number -= numeralMap[i].value;
+        }
+      }
+
+      return roman;
+    }
+  </script>
```
Comment (ahzamir) [inline index.html]: - You need to have your JS code in a separate file like `script.js` and add its link using `<script>` tag at the end of your <body>` in your HTML file.

File: README.md [hunk 3]
```diff
@@ -1,2 +1,22 @@
...
+- CSS
+- JavaScript
+
+## Author 👩‍💻
+
+- LinkedIn: https://www.linkedin.com/in/sumita-balouch-959197309?utm_source=share&utm_campaign=share_via&utm_content=profile&utm_medium=ios_app
+- Email: sumitabalouch@gmail.com
+
+## License 📜
+MIT license.
```
Comment (ahzamir) [inline README.md]: - Add a demo section with a screenshot of your project and also add How to use section to your README.md file

**Review by ahzamir — CHANGES_REQUESTED** [review]
@S-BLH 

- Well done you have implemented some of the requirements but still your project needs some changes please check the comments below for more details:

**Comment by S-BLH** [comment]
@ahzamir ostad I solved the probs, please review it.

**Comment by ahzamir** [comment]
@S-BLH 

- Your demo image is not showing in your README file
  - Inorder to add the screenshot to your readme you have to option:
    - You can add the screenshot to your project folder and linked it.
    - You can drag and drop it directly yo your readme in githib.

**Comment by S-BLH** [comment]
@ahzamir please kindly, re-review my project, as I modified the errors

File: README.md [hunk 4]
```diff
@@ -1,3 +1,26 @@
-# S-BLH-roman-numeral-converter
-2nd project of java script
+<<<<<<< HEAD
```
Comment (ahzamir) [inline README.md:1]: - Remove this

File: README.md [hunk 5]
```diff
@@ -1,3 +1,26 @@
...
+## Technologies Used 🛠️
+- HTML
+- CSS
+- JavaScript
+
+## Usage 🎯
+to use this application, you only have to write the greek numbers and then click on the convert, to change them to roman numbers
+
+## Demo 📸
+![Demo]:C:\Users\Black Store\Desktop\Screenshot 2024-06-30 144151.png
```
Comment (ahzamir) [inline README.md]: - Here, it is not showing the screenshot, I told you not to use image link from your Disk C or ... instead, add the screenshot to your project folder and then use its link like this `FILE_NAME.FILE_FORMAT` for example `image.png`

File: README.md [hunk 6]
```diff
@@ -1,3 +1,26 @@
...
+live link of the project:https://s-blh.github.io/S-BLH-roman-numeral-converter/
```
Comment (ahzamir) [inline README.md]: Change this like this:
- Just copy and paste this
```
[live link of the project](https://s-blh.github.io/S-BLH-roman-numeral-converter/)
```

**Review by ahzamir — CHANGES_REQUESTED** [review]
@S-BLH 

- Still your README.md file is not clean please don't submit with the same mistake again and again and don't make changes to your development branch as it will create conflicts.

**Comment by S-BLH** [comment]
@ahzamir sorry ostad I didnt know that the screen shot  linkdidnt work, please review it.

File: README.md [hunk 7]
```diff
@@ -1,3 +1,25 @@
...
+## Technologies Used 🛠️
+- HTML
+- CSS
+- JavaScript
+
+## Usage 🎯
+to use this application, you only have to write the greek numbers and then click on the convert, to change them to roman numbers
+
+## Demo 📸
+![Demo]:![alt text](<Screenshot 2024-06-30 144151-1.png>)
```
Comment (ahzamir) [inline README.md]: - I can not see your screenshot in your project folder and also you used here wrong use it like this:
- First, add your screenshot to your project folder
- Then just copy and paste this and add the name of your screenshot:
```
![Image](HERE_REMOVE THE TEXT AND ADD YOUR OWN IMAGE NAME AND FORMAT EXAMPLE screenshot.png)
```

**Review by ahzamir — CHANGES_REQUESTED** [review]
@S-BLH 

- Still got the same issue with the screenshot

**Review by ahzamir — APPROVED** [review]
### Hi @S-BLH 🙋‍♀️,

> Excellent work! 💯 You have done a great job implementing all the project requirements. 🥳 Your feature is complete! Congratulations!!!🎊

- **[REQUIRED]** Share your project on this [GROUP](https://www.facebook.com/groups/kick-startdev), add a simple Proud Moment text and a screenshot of your project then besides that add your project link at the end of the text as a GitHub Source code reference and post.

- Share your Project on your LinkedIn account and add a proud moment text on it with a screenshot.

- Well Done great design Implemented. 🎯🎉👍
image

# Status: Approved ✅
There is nothing to say other than to merge your branch. 🎉🎯✅👍😊🥳

WinkThumbsUpGIF

 ## Highlights 🌞

🎉 Great Design Implemented 
✔️ Created a professional Repo (You created a repository with your project name)
✔️ Correct use of Gitflow (You created the development branch and made it your default branch, then created and committed your work on your feature branch)
✔️ Commit messages are meaningful
✔️ Setup the linters correctly (You added the correct linters config into your project according to the language you used on your code)
✔️ No linter errors (All linters checks have been passed 🟢)
✔️ You have followed the Roman Numeral Converter project task and committed using the command line
✔️ Created the pull request (From your feature branch to the development branch you created a pull request and submitted the pull request link for review)
✔️ Good PR description (You added a meaningful description to your PR about your work and the features you added on this pull request)
✔️Your code is clean
✔️Your repo has a professional README.md file
✔️Your repo have been reviewed by your partner
✔️Your design is responsive

**Cheers 🥂 and Happy coding!!! 👯 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me @ahzamir in your question so I can receive the notification.**

**Pull request merged**